src/components/panel/panel.js

```diff
@@ -1306,10 +1386,22 @@ MdPanelRef.prototype._updatePosition = function(init) {
       this.panelContainer.addClass(MD_PANEL_HIDDEN);
     }
 
-    this.panelEl.css(MdPanelPosition.absPosition.TOP, positionConfig.getTop());
-    this.panelEl.css(MdPanelPosition.absPosition.BOTTOM, positionConfig.getBottom());
-    this.panelEl.css(MdPanelPosition.absPosition.LEFT, positionConfig.getLeft());
-    this.panelEl.css(MdPanelPosition.absPosition.RIGHT, positionConfig.getRight());
+    this.panelEl.css(
```

Comment:
@bradrich I wouldn't right now. It's going to cause too many conflicts and it's wrong at 80. Additional benefit of 80 is that I can use formatting tools, which I can't if the file is 100 (because it would force it back to 80).